modules/openapi-generator/src/main/resources/JavaSpring/libraries/spring-boot/pom.mustache

```diff
@@ -134,6 +134,28 @@
             <groupId>javax.validation</groupId>
             <artifactId>validation-api</artifactId>
         </dependency>
-{{/useBeanValidation}}        
+{{/useBeanValidation}}
+{{#virtualService}}
+<!-- START Virtual Service API support -->
+		<dependency>
+			<groupId>org.openapitools</groupId>
+			<artifactId>openapi-virtualan</artifactId>
```

Comment:
Is it correct to say that you also want to host this project under "org.openapitools"?

In other words, we'll need to move the project from https://github.com/elans3/openapi-virtualan to https://github.com/openapitools/openapi-virtualan, right?

Reply:
@wing328 : yes this was my personal suggestion. I think OpenAPITools organization can have other projects on board.

But this is something that needs to be validated by the team.

Reply:
@jmini - I agree with you.. Just brought that just we had started with the discussion :)